v1/test/e2e/lib/util.sh

```
@@ -45,6 +45,27 @@ function cleanup_conn() {
     [[ -n $PID_CONN ]] && kill $PID_CONN
 }
 
+function download_binary() {
+    url=$0
+    name=$1
+
+    max_attempts=3
+    attempt=1
+    while [[ $attempt -le $max]]; do
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this while loop.

(SC1009)